Makefile

```diff
@@ -47,29 +72,3 @@ copyright:
 coverage: $(generated)
 	$(PYTHON) -m coverage run --source myhoard -m pytest $(PYTEST_ARG) test/
 	$(PYTHON) -m coverage report --show-missing
-
-.PHONY: flake8
-flake8: $(generated)
-	$(PYTHON) -m flake8 --exclude=__init__.py --max-line-len=125 $(ALL_PYTHON_DIRS)
-
-.PHONY: isort
-isort:
-	time isort --recursive $(ALL_PYTHON_DIRS)
-
-.PHONY: reformat
-reformat: isort yapf
-
-.PHONY: test
-test: flake8 pylint copyright unittest
```

Comment:
Copyright target is no longer executed in tests at all but would seem relevant.

Reply:
That's an oversight, I'll reinstate it.